### PR TITLE
DelegatedHaxballRoom: added dynamic method integration

### PR DESCRIPTION
**Pull request by ghost**
Added dynamic method integration and removed static one.

### CODE REVIEW

File: src/delegated-haxball-room.js [hunk 1]
```diff
@@ -36,6 +36,12 @@ export default class DelegatedHaxballRoom {
             this._room = HBInit({ ...config, roomName, playerName, maxPlayers, isPublic, geo })
 
             this._delegateCallbacks()
+
+            // integrate original RoomObject methods to framework
```
Comment (olehmisar) [inline src/delegated-haxball-room.js]: Comments should start with capital letter and end with full stop (.).

File: src/delegated-haxball-room.js [hunk 2]
```diff
@@ -36,6 +36,12 @@ export default class DelegatedHaxballRoom {
...
+            Object.getOwnPropertyNames(this._room).forEach(prop => {
+              if(typeof this._room[prop] == "function")
```
Comment (olehmisar) [inline src/delegated-haxball-room.js]: Use `_.isFunction` instead of `typeof`.

**Comment by olehmisar** [comment]
You should also make changes in `haxilium.js` file because I use `super` keyword there. You have to replace `super` with `this._room`.

**Comment by ghost** [comment]
I don't think I need to replace super, because i assigned those methods to the class itself.

**Comment by ghost** [comment]
You can checkout to that commit and see if it works.

**Comment by olehmisar** [comment]
Thanks, merged. Your way of delegating methods is very cool.

**Pull request merged**